lib/nunjucks/views/manage-prototype/scripts.njk

```diff
@@ -1,3 +1,4 @@
-<script src="/manage-prototype/dependencies/govuk-frontend/govuk/all.js"></script>
-<script src="/manage-prototype/dependencies/govuk-frontend/govuk-prototype-kit/init.js"></script>
+{% for script in govukFrontend.scripts -%}
+<script src="/manage-prototype/dependencies/govuk-frontend{{ script }}"></script>
+{% endfor %}
```

Comment:
There's a better implementation of this in the PR from @BenSurgisonGDS's
[https://github.com/alphagov/govuk-prototype-kit/pull/2319/files](https://github.com/alphagov/govuk-prototype-kit/pull/2319/files#diff-a204f6f327491cbf3267b7d38f719a8161f2fbe416817d1ed7d5d5ab25cfcb1cR1-R7)

Will need to make sure `managePrototypeMiddleware()` loads the files from `assetPath`

Great to see the two come together

Reply:
As discussed on Slack, we're going to keep these hard coded